eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -559,4 +561,12 @@ class EclairImpl(appKit: Kit) extends Eclair with Logging {
       case Attempt.Failure(cause) => Future.successful(SendOnionMessageResponse(sent = false, failureMessage = Some(s"the `content` field is invalid, it must contain encoded tlvs: ${cause.message}"), response = None))
     }
   }
+
+  override def stop(exitCode: Int): Future[Unit] = {
```

Comment:
I wouldn't have provided the `exitCode` as parameter. We should use `0` as this is a graceful stop.

Reply:
done in https://github.com/ACINQ/eclair/pull/2233/commits/974a73ce24972f6caac7d9dc31e3d2a4b0da24af